omero/sysadmins/unix/server-install-homebrew.txt

```diff
@@ -209,70 +212,45 @@ If you encounter problems with the installation script, please take a look at
 Configuration
 -------------
 
-Environment variables
-^^^^^^^^^^^^^^^^^^^^^
-
-Edit your :file:`.profile` as appropriate. The following are indicators of
-required entries and correspond to a Homebrew installation of OMERO |release|:
-
-::
-
-    export PYTHONPATH=$(brew --prefix omero)/lib/python
-    export PATH=/usr/local/bin:/usr/local/sbin:/usr/local/lib/node_modules:$PATH
-
-.. note::
-    If you have a local :file:`.bash_profile` file, it will override your
-    :file:`.profile` configuration file.
-
-.. note::
-    On Mac OS X Lion, a version of PostgreSQL is already installed. If you get an error like the following:
-
-    ::
-
-        psql: could not connect to server: Permission denied
-        Is the server running locally and accepting
-        connections on Unix domain socket "/var/pgsql_socket/.s.PGSQL.5432"?
-
-    make sure :file:`/usr/local/bin` is at the beginning of your
-    :envvar:`PATH` (see also `this post <http://nextmarvel.net/blog/2011/09/brew-install-postgresql-on-os-x-lion/>`_).
-
-
 Database creation
 ^^^^^^^^^^^^^^^^^
 
-Start the PostgreSQL server.
-If you wish to use ``brew services`` to start postgresql, you will first need to 
-install it:
+#. Create a new database cluster::
 
-::
+    $ initdb -E UTF8 /usr/local/var/postgres
 
-    $ brew tap gapple/services
+#. Start the PostgreSQL server.
 
-::
+  - If you wish to use ``brew services`` to start postgresql, you will
```

Comment:
These bullet points are giving the stupid plone highlighting in staging, although I'm not sure whether you can manage to get bullet points within a numbered section otherwise… I'll have a play in a bit